kolibri/utils/server.py

```diff
@@ -580,13 +609,26 @@ def start(port=0, zip_port=0, serve_http=True, background=False):
         autoreloader = Autoreloader(bus)
         autoreloader.subscribe()
 
+    reload_plugin = RestartPlugin(bus)
```

Comment:
I find `RestartPlugin` a confusing name. Reading plugins in the kolibri context does not bring magicbus to my mind at first

Reply:
Can update - but there are lots of classes called Plugin in this file!